src/libm-tester/CMakeLists.txt

```diff
@@ -27,6 +27,28 @@ if(COMPILER_SUPPORTS_SSE2)
   list(APPEND IUT_LIST ${TARGET_IUTSSE2})
 endif(COMPILER_SUPPORTS_SSE2)
 
+if(COMPILER_SUPPORTS_SSE4)
+  add_executable(${TARGET_IUTSSE4} iutsimd.c iutsimdmain.c testerutil.c)
+  target_compile_options(${TARGET_IUTSSE4}
+    PRIVATE ${FLAGS_ENABLE_SSE4})
+  target_compile_definitions(${TARGET_IUTSSE4}
+    PRIVATE ENABLE_SSE4=1)
+  target_link_libraries(${TARGET_IUTSSE4} ${LIB_SLEEF} ${EXTRA_LIBS})
+
+  list(APPEND IUT_LIST ${TARGET_IUTSSE4})
```

Comment:
This, and all the similar lines appending testers to the IU_LIST variable, should be guarded by different variables other then `COMPILER_SUPPORT_*`. The fact is that a target might not have the vector extension that the compiler has been able to compile, so that the actual `IUT_LIST` should be updated by a `TARGET_RUNS_*`, which should be produced out  of `try_run` instead of `try_compile`.

The code is working as it is because the `tester` does not report a failure if the `iut*` that is being executed is not supported by the target. It is fine for me to leave this as it is, but I would like to hear @shibatch 's opinion.